### ISSUE TITLE
Why doesn't the official theme allow for bugs to be submitted？

### ISSUE THREAD

**Issue by linjialiang** [atom/atom#18153]
![default](https://user-images.githubusercontent.com/8848263/46238869-6aa46d80-c3c3-11e8-9f51-eed21aa1f600.png)

**Comment by Aerijo** [comment]
@linjialiang If you are referring to the [old repo](https://github.com/atom/one-dark-ui), it has been migrated here as it says in the README. So this is the place to file the issue.

**Comment by linjialiang** [comment]
> ok ! 
> But there's a problem with win10 if you've seen it:
> - `One Light UI` => Auto-complete prompt image layout without background and center

**Comment by linjialiang** [comment]
> @Aerijo ok ! 
> But there's a problem with win10 if you've seen it:
> - `One Light UI` => auto-complete prompt ico layout with no fixed width or central display

**Comment by Arcanemagus** [comment]
Thanks for reporting this!

This has already been fixed in https://github.com/atom/atom/pull/18016. It is currently scheduled for release when v1.32.0 reaches Stable.

**Comment by linjialiang** [comment]
> @Arcanemagus  @Aerijo There's one more problem, which I've solved here, and you can see if you want to update the next version

> The cause of the problem

![snipaste_2018-09-30_11-04-06](https://user-images.githubusercontent.com/8848263/46252801-95b0bf00-c4a0-11e8-93bc-9d5ede5bf03d.jpg)

> The effect before the solution

![atom](https://user-images.githubusercontent.com/8848263/46252786-4c606f80-c4a0-11e8-95c0-15c4bbd809a1.gif)

> The effect of the solution

![default](https://user-images.githubusercontent.com/8848263/46252787-4ff3f680-c4a0-11e8-93b7-dd22e0b94a8a.gif)

> Custom code has irrelevant information, which I feel works well

```css
// 自定义样式
// 界面字体

project-viewer,
ul[is="atom-tabs"] > li[is="tabs-tab"] > div,
li[is="tree-view-directory"] {
    font-family: 'Source Code Variable', '微软雅黑';
}
// 当前行高亮

.line.cursor-line {
    background-color: rgba(90,138,233,.1);
}

.minimap .cursor-line {
    background-color: rgb(90,138,233);
}
// 目录树

ul[location="left"][is="atom-tabs"] {
    display: none;
}

atom-pane > ul[location="right"],
ol.tree-view-root > li[is="tree-view-directory"] > div.header {
    z-index: 3;
    font-size: 1.3rem;
    height: 3.5rem;
    line-height: 3.5rem;
}

atom-pane > ul[location="right"] > li.active,
ol.tree-view-root > li[is="tree-view-directory"] > div.header {
    border-bottom: 2px solid rgba(90,138,233, .3);
}

ol.list-tree > li[is="tree-view-directory"] div.list-item:not(.project-root-header),
ol.list-tree li[is="tree-view-directory"]::before,
ol.list-tree li[is="tree-view-file"],
ol.list-tree li[is="tree-view-file"]::before,
 {
    height: 2.5rem;
    line-height: 2.5rem;
    margin: 0 !important;
    padding: 0 !important;
}

ol.list-tree li[is="tree-view-file"] {
    margin-left: 17px!important;
}

ul[location="center"][is="atom-tabs"].list-inline.tab-bar.inset-panel {
    height: 3.5rem;
}

ul[location="center"][is="atom-tabs"].list-inline.tab-bar.inset-panel > li[is="tabs-tab"][data-type="TextEditor"],
ul[location="center"][is="atom-tabs"].list-inline.tab-bar.inset-panel > li[is="tabs-tab"][data-type="SettingsView"] {
    line-height: 3.5rem;
}

ul[location="center"][is="atom-tabs"].list-inline.tab-bar.inset-panel > li[is="tabs-tab"][data-type="TextEditor"].active {
    border-bottom: 2px solid rgba(90,138,233,.3);
    transition: background-color 1s, color 1s;
}

ul[location="center"][is="atom-tabs"].list-inline.tab-bar.inset-panel > li[is="tabs-tab"][data-type="TextEditor"] {
    transition: background-color 1s, color 1s;
}

ul[location="right"] .close-icon,
ul[location="center"][is="atom-tabs"].list-inline.tab-bar.inset-panel .close-icon {
    top: 0!important;
    bottom: 0;
    margin: auto 0;
}

ul[location="right"] .close-icon {
    font-size: 1rem!important;
    line-height: 3.5rem;
}

ul[location="center"][is="atom-tabs"].list-inline.tab-bar.inset-panel .close-icon:hover {
    transition: background-color 0.3s, color 0.3s;
}

ul[location="center"][is="atom-tabs"].list-inline.tab-bar.inset-panel .close-icon {
    transition: background-color 0.3s, color 0.3s;
}

ul[location="right"] .close-icon:hover,
ul[location="right"] .close-icon:active,
ul[location="center"][is="atom-tabs"].list-inline.tab-bar.inset-panel .close-icon:hover,
ul[location="center"][is="atom-tabs"].list-inline.tab-bar.inset-panel .close-icon:active {
    cursor: pointer;
}

.tree-view .selected:before,
li.list-item.selected:focus::before {
    background-color: transparent!important;
}

li[is="tree-view-directory"].selected > .header,
li[is="tree-view-file"].selected {
    background: rgba(90,138,233, .6);
    height: 3rem;
    line-height: 3rem;
}

li[is="tree-view-directory"]:not(.selected) > .header:not(.project-root-header):hover,
li[is="tree-view-file"]:not(.selected):hover {
    background: rgba(90,138,233, .3);
    height: 3rem;
    line-height: 3rem;
}

li[is="tree-view-directory"].selected > .header:not(.project-root-header) span,
li[is="tree-view-directory"].selected > .header:not(.project-root-header) span::before,
li[is="tree-view-directory"]:not(.selected) > .header:not(.project-root-header):hover,
li[is="tree-view-file"].selected span,
li[is="tree-view-file"].selected span::before,
li[is="tree-view-file"]:not(.selected):hover,
li[is="tree-view-file"]:not(.selected):hover span::before {
    font-size: 1.2em;
    color: white;
    transition: background-color 0.3s, color 0.3s, font-size 0.3s;
}

li[is="tree-view-file"] span,
li[is="tree-view-file"] span::before,
li[is="tree-view-directory"] .header:not(.project-root-header) span {
    transition: font-size 0.3s;
}
// 项目列表

project-viewer h2.heading {
    font-size: 0;
    border-bottom: 2px solid rgba(90,138,233,.3);
    height: 3.5rem;
    margin: 0 auto;
    position: relative;
    text-align: center;
    line-height: 3.5rem;
    padding: 0;
}

project-viewer h2.heading::before {
    content: '项目列表';
    font-size: 1.3rem;
    position: absolute;
    left: 0;
    right: 0;
}

project-viewer div.body-content {
    overflow: hidden;
}

project-viewer div.body-content > ul {
    margin: 0;
}

project-viewer li[is="project-viewer-project"],
project-viewer li.selected[is="project-viewer-project"]::before {
    line-height: 3rem;
    height: 3rem;
}

project-viewer li[is="project-viewer-project"] > span,
project-viewer li[is="project-viewer-project"] > span::before {
    font-size: 1.3em!important;
    line-height: 3rem;
}

project-viewer li[is="project-viewer-project"].list-item:not(.selected):not(.no-paths)::before {
    height: 3rem;
}

```

**Comment by Arcanemagus** [comment]
If I'm understanding you correctly, that was also already fixed in https://github.com/atom/one-light-ui/pull/140, which was released in v1.31.0. Since your screenshot lists `one-dark-ui` as v1.12.4 and not v1.12.5 I'm assuming you haven't updated yet. You didn't fill out the template though so we can't really know for sure.

In the future please make sure your Atom is up to date and you fill out the template so we can see issues like that sooner.

Thanks!
